### ISSUE TITLE
Sequence/structure analysis

### ISSUE THREAD

**Issue by sillitoe** [UCLOrengoGroup/cath-todo#19]
For 3.20.20.190 (and hopefully 2.30.29.30) provide:
 * sequence alignments (based on structure)
 * structure superpositions (including HETATMs)

**Comment by tonyelewis** [comment]
Attaching [3.20.20.190.zip](https://github.com/UCLOrengoGroup/cath-todo/files/1111291/3.20.20.190.zip) (GitHub's preferred format) containing files arising from aligning+superposing the s-reps of [3.20.20.190](http://www.cathdb.info/version/v4_1_0/superfamily/3.20.20.190) [v4.1.0]:
 * **`3.20.20.190.faa`** - a FASTA alignment
 * **`3.20.20.190.html`** - an HTML rendering of the alignment:
   * rainbow colouring (blue &rarr; red) shows the position on the alignment (N &rarr; C)
   * colour saturation shows structural similarity
 * **`3.20.20.190.pml`** - a PyMOL superposition:
   * F1 &ndash; F3 keys change the colouring scheme
   * black sticks are [atoms in non-polymer organic compounds](https://pymolwiki.org/index.php/Single-word_Selectors), accessible under the `organic` selection
   * `core`/`noncore` selections identify structurally well-conserved/poorly-conserved residues respectively (so, for example, clicking `noncore` &rarr; `H` (Hide) &rarr; `everything` leaves only the core showing)
   * `alignment` shows lines between aligned residues and can be activated by selecting `alignment` and then clicking `S` (Show) &rarr; `dashes` (note: the distorted cartoon representation can miss the alignment lines whereas the ribbon representation touches them precisely)
 * **`3.20.20.190.superposed_pdbs.tar.gz`** - a `.tar.gz` of a directory of the superposed PDB files

These files should contain all residues and all pertinent `ATOM` / `HETATM` records (including, eg `MSE` `HETATM` records). Please shout if you spot any absences.

Here's an image of the superposition: ![3 20 20 190](https://user-images.githubusercontent.com/9743330/27678228-67558fa8-5cac-11e7-83a6-005cf190af41.jpg)

**Comment by tonyelewis** [comment]
At present, cath-tools's high-quality alignment procedure is a bit slow to handle the 116 s-reps in [2.30.29.30](http://www.cathdb.info/version/v4_1_0/superfamily/2.30.29.30) but I'm happy to generate derived files if someone gives me an alignment.

**Comment by nathaliereuter** [comment]
Dear Tony and Ian,

would it be possible to obtain the same types of alignments (structure and sequence) for the superfamily of PH domains, i.e. Superfamily 2.30.29.30? It is a fairly big family with many structure representatives....

Best,
Nathalie

**Comment by tonyelewis** [comment]
We haven't been able to look into vastly speeding up the heavy-refining process we previously used (above) but we have added a light-refining process. This can leave the alignment more gappy than it should be (ie it doesn't always align residues that should be aligned) but it should do a pretty good job of aligning core residues.

I've performed this light-refining structural alignment for 2.30.29.30 with v4.2.0 data. You can get the results from this file : [2.30.29.30.4_2_0.light_refining.tar.gz](https://github.com/UCLOrengoGroup/cath-todo/files/1934236/2.30.29.30.4_2_0.light_refining.tar.gz).

For a short time, I'll also leave the individual files on my account on the UCL server:
 * [2.30.29.30.4_2_0.light_refining.fa](http://www.biochem.ucl.ac.uk/~ucbctnl/2.30.29.30.4_2_0.light_refining.fa) - FASTA alignment
 * [2.30.29.30.4_2_0.light_refining.html](http://www.biochem.ucl.ac.uk/~ucbctnl/2.30.29.30.4_2_0.light_refining.html) - HTML rendering of alignment
 * [2.30.29.30.4_2_0.light_refining.pml](http://www.biochem.ucl.ac.uk/~ucbctnl/2.30.29.30.4_2_0.light_refining.pml) - PyMOL script containing superposition (*very* slow to load)
 * [2.30.29.30.4_2_0.light_refining.pymol_draw.jpg](http://www.biochem.ucl.ac.uk/~ucbctnl/2.30.29.30.4_2_0.light_refining.pymol_draw.jpg) PyMOL image of the superposition

**Comment by nathaliereuter** [comment]
Thank you very much Tony, this is extremely helpful. The extra gaps in the sequence alignment should not be much of a problem for us, we'll have a look at it.

**Comment by hkabbech** [comment]
Dear Tony,

Thank you for the files. I'm working with Nathalie on a project based on these structural alignments.
Please, may you add the directory of the superposed PDB files for the 2.30.29.30 superfamily ?

Kind regards,
Helene

**Comment by tonyelewis** [comment]
Yes - here are the PDBs: [2.30.29.30.4_2_0.light_refining.pdbs.tar.gz](https://github.com/UCLOrengoGroup/cath-todo/files/1934976/2.30.29.30.4_2_0.light_refining.pdbs.tar.gz)

...and here's a PyMOL ray-traced image of the superposition: 
![2 30 29 30 4_2_0 light_refining pymol_ray](https://user-images.githubusercontent.com/9743330/39088496-1af796d6-45ab-11e8-809d-4eee6fb6da0a.jpg)

**Comment by nathaliereuter** [comment]
Thank you!
